### DISCUSSION TITLE
data not importing correctly into table

### DISCUSSION THREAD

**Discussion by achau99** [pawelsalawa/sqlitestudio#5431] [General]
### Details
i am importing a CSV table into a database as a table, and my table has 107k rows. when i import it, i only have 92k rows and it says the import was successful. i am not sure whats happening

### Steps to reproduce
import table, choose CSV, then import. data does not all show up in the import

**Comment by pawelsalawa** [comment]
How do you know your CSV has 107k rows? I mean 1 line  of CSV not necessarily means 1 row. CSV can use quote for multi-line values, in which case you will have more lines in CSV than actually proper rows.
